src/Api/Fields/ReportApiFields.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Silktide\ProspectClient\Api\Fields;
+
+use DateTime;
+use Psr\Http\Message\UriInterface;
+
+class ReportApiFields extends AbstractApiFields
```

Comment:
Naming-wise, Request feels more obvious than "Fields" - unless I've misunderstood the purpose of this class

Reply:
The ApiFields represent the key-value pairs that are sent when creating new reports. I used the term "Fields" because of the API docs referring to them as fields when they are custom. They are also referred to as properties in the API docs.